gems/pending/openstack/openstack_handle/handled_list.rb

```diff
@@ -0,0 +1,118 @@
+module OpenstackHandle
+  module HandledList
+    def handled_list(collection_type, options = {})
```

Comment:
Naming is hard...but, I'm not crazy about this name (method or module).

I think I'd rather see:

```ruby
module OpenstackResourceList
  def list(collection_type, options = {})
...
```